src/frontend/apps/standalone_site/src/App.tsx

```diff
@@ -37,17 +39,19 @@ const App = () => {
         throw err;
       }}
     >
-      <Grommet theme={themeExtend}>
-        <BrowserRouter>
-          <Authenticator>
-            <Main height={{ min: '100vh' }}>
-              <Page kind="full">
-                <AppRoutes />
-              </Page>
-            </Main>
-          </Authenticator>
-        </BrowserRouter>
-      </Grommet>
+      <QueryClientProvider client={queryClient}>
+        <Grommet theme={themeExtend}>
+          <BrowserRouter>
+            <Authenticator>
+              <Main height={{ min: '100vh' }}>
+                <Page kind="full">
+                  <AppRoutes />
+                </Page>
+              </Main>
+            </Authenticator>
+          </BrowserRouter>
+        </Grommet>
+      </QueryClientProvider>
```

Comment:
Don't you want to add the `ReactQueryDevtools` to help debug?
```suggestion
      <QueryClientProvider client={queryClient}>
        <ReactQueryDevtools />
        <Grommet theme={themeExtend}>
          <BrowserRouter>
            <Authenticator>
              <Main height={{ min: '100vh' }}>
                <Page kind="full">
                  <AppRoutes />
                </Page>
              </Main>
            </Authenticator>
          </BrowserRouter>
        </Grommet>
      </QueryClientProvider>
```

Reply:
I know you will miss it, I will add it tho <3